src/Playwright.NUnit/ContextTest.cs

```diff
@@ -41,4 +41,11 @@ public async Task ContextSetup()
     {
         Context = await NewContext(ContextOptions()).ConfigureAwait(false);
     }
+
+    [TearDown]
+    public async Task ContextTearDown()
+    {
+        await Context.CloseAsync().ConfigureAwait(false);
+        Context = null!;
```

Comment:
I could not find why this is done, and it seems a bit obscure to force break your nullability. Are you checking it for null somewhere, or are you just trying to get it garbage collected earlier?